src/Mbstring/Mbstring.php

```diff
@@ -392,6 +392,8 @@ public static function mb_encoding_aliases($encoding)
             case 'UTF8':
             case 'UTF-8':
                 return array('utf8');
+            case 'ASCII':
+                return array('cp367', 'csASCII', 'iso-ir-6', 'IBM367', 'IBM-367');
```

Comment:
Is that really true? I'm not sure providing any of these would make the polyfill act as if 'ascii' was provided. This would deserve some tests.

Reply:
but this has a technical meaning, we don't want to list aliases that don't work
these don't work, so we don't list them.